src/lib/services-manager/dialogs/services_manager.rb

```diff
@@ -177,6 +181,68 @@ def buttons
         )
       end
 
+      # Help of dialog buttons
+      #
+      # @return [String]
+      def buttons_help
+        # TRANSLATORS: help section to describe the dialog buttons
+        _("<h2>Buttons description:</h2>") +
+          start_stop_button_help +
+          start_mode_button_help +
+          logs_button_help +
+          show_details_button_help +
+          apply_button_help +
+          ok_button_help +
+          "<br />" +
+          # TRANSLATORS: note about dialog behavior
+          _("Note: the services will not be modified until all changes are saved by using 'Apply' or 'OK' button.")
```

Comment:
"by using **the** 'Apply' or **the** 'OK' button"

Yes, I know, taken from the old code, but still broken English :smile: